CONTRIBUTING.md

```diff
@@ -63,13 +63,30 @@ We use a centralized approach that leaves `main` as the current development vers
 
 ## Licenses and Copyrights
 
+### Plain English
+
+Legal language can be tricky. Here is what our legal terms are, in plain English. Anywhere that these plain English terms conflict with the legal terms, the legal terms take precedence.
+
+* When you contribute code, you still own the code, but we are free to use it forever.
+* Contributing code to the project does not automatically mean that the end product will be suitable for a particular purpose you may have.
+* If your contribution causes us harm, and you could have prevented it, you are responsible for the harm.
```

Comment:
This is an interesting one. I wonder if people will take on such a burden. One way to prevent the harm is not to contribute. But it's clear at least, and that I appreciate.

Reply:
It's a pretty standard clause. I could modify it to specify what we're targeting here is intellectual property violations that you could have, and should have avoided. That's what the actual legalese says and I could express that kind of limit here.

Reply:
Yes please. If someone leaves a project can they be retroactively liable too? Is there such a thing as open source contributors insurance?